### PR TITLE
Add a manifest for AWS Config Service

### PR DESCRIPTION
**Pull request by jeskew**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,8 +4,8 @@ build/artifacts
 phpunit.xml
 phpunit.functional.xml
 test_services.json
-Config
-./Makefile
+/Config
```
Comment (GrahamCampbell) [inline .gitignore]: ?

File: .gitignore [hunk 2]
```diff
@@ -4,8 +4,8 @@ build/artifacts
...
+/Makefile
```
Comment (GrahamCampbell) [inline .gitignore]: ?

**Comment by jeremeamia** [comment]
Awesome. You're official now. :ship:

File: .gitignore (hunk 1 above)
Comment (jeskew) [inline .gitignore]: This line was meant to exclude a file at the project root named `Config` but was also excluding a folder from the AWS manifests for a service named `config`.

Comment (GrahamCampbell) [inline .gitignore]: Ah right. :)

File: .travis.yml [hunk 3]
```diff
@@ -25,4 +25,4 @@ install:
 
 script:
   - make test
-  - [ -z "$COMPOSER_OPTS" ] && make package
```
Comment (mtdowling) [inline .travis.yml]: Why this change? Was it not working before?

Comment (jeskew) [inline .travis.yml]: Travis was failing to interpret the manifest file and wasn't testing any new builds.

Comment (mtdowling) [inline .travis.yml]: Ah I see.
